server/backend/README.md

```
@@ -73,3 +73,60 @@ The full environment-variable table for self-hosters lives in
 
 [issue-311]: https://github.com/mozilla-ai/cq/issues/311
 [issue-312]: https://github.com/mozilla-ai/cq/issues/312
+[issue-310]: https://github.com/mozilla-ai/cq/issues/310
```

markdownlint-cli2 (0.22.1)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "issue-310"

(MD053, link-image-reference-definitions)